<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encrypter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTableView" name="filesList"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QPushButton" name="cryptAll">
        <property name="text">
         <string>Crypter</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="decryptAll">
        <property name="text">
         <string>Décrypter</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="remove">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="importButton">
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="action_retrieveKey"/>
    <addaction name="action_checkForUpdates"/>
    <addaction name="action_openSettings"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="action_newKey"/>
    <addaction name="action_saveKey"/>
    <addaction name="action_saveSessionKey"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_retrieveKey">
   <property name="text">
    <string>Obtenir la clé</string>
   </property>
  </action>
  <action name="action_openSettings">
   <property name="text">
    <string>Paramètres</string>
   </property>
  </action>
  <action name="action_checkForUpdates">
   <property name="text">
    <string>Vérifier les mises à jour</string>
   </property>
  </action>
  <action name="action_saveKey">
   <property name="text">
    <string>Sauvegarder la clé</string>
   </property>
  </action>
  <action name="action_newKey">
   <property name="text">
    <string>Nouvelle clé</string>
   </property>
  </action>
  <action name="action_saveSessionKey">
   <property name="text">
    <string>Créer la clé de session</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
